src/pkg/projects/netcoreapp/src/localnetcoreapp.override.targets

```diff
@@ -4,34 +4,47 @@
   <Target Name="OverrideRuntimeFilesFromPackageResolve" Condition="'$(CoreCLROverridePath)' != ''" BeforeTargets="GetFilesFromPackageResolve">
     <Error Condition="!Exists('$(CoreCLROverridePath)')" Text="The path provided to CoreCLROverridePath ($(CoreCLROverridePath)) does not exist." />
     <PropertyGroup>
-      <CoreCLRPDBOverridePath Condition="'$(CoreCLRPDBOverridePath)' == '' and Exists('$(CoreCLROverridePath)/PDB')">$(CoreCLROverridePath)/PDB</CoreCLRPDBOverridePath>
+      <CoreCLRCrossTargetComponentFolderName Condition="'$(TargetArchitecture)' == 'arm64' and '$(BuildArchitecture)' != 'arm64'">x64</CoreCLRCrossTargetComponentFolderName>
+      <CoreCLRCrossTargetComponentFolderName Condition="'$(TargetArchitecture)' == 'arm' and '$(BuildArchitecture)' != 'arm' and '$(TargetsWindows)' == 'true'">x86</CoreCLRCrossTargetComponentFolderName>
+      <CoreCLRCrossTargetComponentFolderName Condition="'$(TargetArchitecture)' == 'arm' and '$(BuildArchitecture)' != 'arm' and '$(TargetsLinux)' == 'true'">x64</CoreCLRCrossTargetComponentFolderName>
+      <CoreCLRSharedFrameworkFolder>$([MSBuild]::NormalizePath($(CoreCLROverridePath),'sharedFramework'))</CoreCLRSharedFrameworkFolder>
+      <CoreCLRCrossTargetComponentFolder
+        Condition="'$(CoreCLRCrossTargetComponentFolderName)' != ''">$([MSBuild]::NormalizePath($(CoreCLROverridePath),$(CoreCLRCrossTargetComponentFolder),'sharedFramework'))</CoreCLRCrossTargetComponentFolder>
```

Comment:
It's odd that these two dirs are treated as file paths (normalized without trailing `/` rather than using `NormalizeDirectory`) and the names don't have any of the conventional `Path`, `File`, `Dir`, or `Directory` suffixes. (At least... those are the conventions I'm aware of.) Is there a reason for this?

Reply:
I'll change the naming.

Reply:
There wasn't any particular reason.